extension/src/components/ConfirmLoanWizard.tsx

```diff
@@ -51,9 +52,7 @@ const ConfirmLoanWizard = ({ onCancel, onSuccess, loanToSign }: ConfirmLoanWizar
             const loanBackup = await createLoanBackup(signedTransaction);
             const file = new Blob([JSON.stringify(loanBackup)], { type: "text/json" });
             const url = URL.createObjectURL(file);
-            // Note: it would be nicer to open a dialog to download the file. However, we would lose focus
-            // of the window. We don't want that, hence we just open a new tab with the content
-            window.open(url, "_blank");
```

Comment:
Nice!

This commit is candidate for squashing into the earlier one!

Reply:
I can recommend looking into https://github.com/tummychow/git-absorb for a semi-automated solution to this.